WMCDuplicateRemover.Tests/EPG/EPGIntegrationTests.cs

```diff
@@ -17,7 +13,9 @@ public class EPGIntegrationTests
         [TestFixtureSetUp]
         public void SetUp()
         {
-            tv = new XmlTvEpgWrapper(@"..\..\EPG\TestData\EPG.xml");
+            var dateTimeMock = new Mock<IDateTime>();
```

Comment:
I had some unit tests failing because my test data was out of date. I create an interface to mock out my dependency on DateTime.Now